src/lxc/conf.h

```diff
@@ -74,6 +74,13 @@ struct lxc_cgroup {
 			char *container_dir;
 			char *namespace_dir;
 			bool relative;
+			/* If an unpriv user in pure unified-only hierarchy
+			 * starts a container, then we ask systemd to create
+			 * a scope for us, and create the monitor and container
+			 * cgroups under that.
+			 * This will ignore the above things like monitor_dir
+			 */
+			char *systemd_scope;
```

Comment:
Seems like we probably need to free this in lxc_conf_free() (or make a helper to free cgroup_meta and call that)?

Reply:
Oh that should be done in lxc_conf_free(), it looks like.  Thanks.